.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -18,6 +18,7 @@ labels: enhancement
 <!-- Provide a list of conditions that must be satisfied to close this issue. -->
 - [ ] Make sure documentation is up-to-date, create one if needed
 - [ ] Add test coverage for happy-path (unit or e2e test)
+- [ ] Record raw demo video for the user-facing changes
```

Comment:
So this will be seen by end user, do you think we need to provide an example demo video if we have any?

Reply:
I don't think we have any example video 🤔 Anyway, for the `help-needed`/`hacktoberfest`/`good-first-issue` we can definitely delete this requirement 👍 This is mostly a reminder for us, active maintainers, so we don't forget about it during estimation and implementation.